src/main/webapp/college-info.html

```diff
@@ -6,15 +6,32 @@
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
   </head>
-  <body>
-    <input type="text" id="school-search"> 
-    <button onclick="getSchoolInfo()">Search</button>
-    <h1 id="school-name"></h1>
-    <h3 id="school-desc"></h3>
-    <div id="students"></div>
-    <div id="race-piechart" style="width: 900px; height: 500px;"></div>
-    <div id="gender-piechart" style="width: 900px; height: 500px;"></div>
-    <div id="admissions"></div>
-    <div id="cost"></div>
+  <body onload="getSchoolInfo()">
```

Comment:
mm, I remembered that `onload=getSchoolInfo()` doesn't work right? @tobaolo Shouldn't we keep the button with `onlick` event here?

Reply:
ahh yes. this got messed up in the merge. i will fix that thanks